tensorflow/python/keras/benchmarks/BUILD

```diff
@@ -109,3 +110,11 @@ py_test(
         "//tensorflow:tensorflow_py",
     ],
 )
+
+py_library(
```

Comment:
Please add it to the `keras_benchmark_lib_pip` library above to make sure it's in the pip dep list.